tools/build_defs/docker/incremental_load.sh.tpl

```diff
@@ -24,8 +24,8 @@ DOCKER="${DOCKER:-docker}"
 
 FULL_DOCKER_VERSION=$(docker version -f {{.Server.Version}} 2> /dev/null \
     || echo "1.10.0")
-DOCKER_MAJOR_VERSION=$(echo "$FULL_DOCKER_VERSION" | sed -r 's#^([0-9]+)\..*#\1#')
-DOCKER_MINOR_VERSION=$(echo "$FULL_DOCKER_VERSION" | sed -r 's#^[0-9]+\.([0-9]+).*#\1#')
+DOCKER_MAJOR_VERSION=$(echo "$FULL_DOCKER_VERSION" | awk -F\. '{ print $1 }')
+DOCKER_MINOR_VERSION=$(echo "$FULL_DOCKER_VERSION" | awk -F\. '{ print $2 }')
```

Comment:
Both awk and sed are in POSIX, and would be expected to be present everywhere bazel would run, including busybox.  Both awk and sed are present in the bazel codebase, with [sed](https://github.com/bazelbuild/bazel/search?q=sed&type=Code) outnumbering [awk](https://github.com/bazelbuild/bazel/search?q=awk&type=Code) by a factor of 4.  

Is awk more portable/stable then sed?  [Hard to say](https://www.google.com/search?q=is+awk+more+portable+than+sed).  What I can say is that the `-r` sed option is not portable, so this is an improvement and IMHO clearer to read.